recipes/lensfunpy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "lensfunpy" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d173e6fdbbeb83c10401ad7af276606e812960307c0d5dc58f03c3a39efd7746
+  patches:
+    - 0002_remove_osx_condition.patch  
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv 
+  skip: True  # [win or py<27]
+   
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config        
+  host:
+    - pip
+    - python
+    - lensfun <0.3.95
+    - numpy
+    - sphinx
+    - wheel >=0.25
+    - enum34  # [py<34]
```

Comment:
Do you need the 3 packages above?

Reply:
They were listed in a `dev-requirements.txt` file so we weren't sure

Reply:
But enum, yes: https://github.com/letmaik/lensfunpy/blob/master/lensfunpy/_lensfun.pyx#L9

Reply:
That's needed at runtime, so no need for it to be in `host`. Only `run` is enough